extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1625,8 +1625,8 @@ export class Project implements ISqlProject {
 		await this.serializeToProjFile(this.projFileXmlDoc!);
 	}
 
-	private async removeFromProjFile(entries: ProjectEntry | ProjectEntry[]): Promise<void> {
-		if (entries instanceof ProjectEntry) {
+	private async removeFromProjFile(entries: IProjectEntry | IProjectEntry[]): Promise<void> {
+		if (!Array.isArray(entries)) {
```

Comment:
Had to change this check slightly since IProjectEntry isn't something we can check with `instanceof` anymore.